dashboard/test/ui/features/support/connect.rb

```diff
@@ -17,6 +17,10 @@ def single_session?
   $browser_config['mobile'] || $single_session
 end
 
+def test_local?
+  return ENV['TEST_LOCAL'] == 'true'
+end
```

Comment:
I appreciate bottlenecking all the `ENV['TEST_LOCAL']` usage through this function. I'd love a comment added to the function what "test_local?" being true means in terms of how tests are run.